lib/ret_web/controllers/api/v1/slack_controller.ex

```diff
@@ -0,0 +1,147 @@
+  def create(conn, params) do
+    %{:query_params => %{"token" => reticulum_token}} = conn
+
+    if reticulum_token !== module_config(:token) do
+      conn
+      |> send_resp(404, "Missing reticulum \"token\" in query params.")
+    else
+      %{
+        "channel_id" => channel_id,
+        # "text" = "help/create/etc. arg1 arg2 arg3"
+        "text" => text,
+        "team_id" => team_id
+      } = params
+
+      # Parse arguments
+      args = String.split(text, " ")
+      command = List.first(args)
+      optional_args = List.delete_at(args, 0)
+
+      handle_command(command, channel_id, optional_args, team_id)
+
+      conn
+      |> send_resp(200, "")
+    end
+  end
+
+  defp handle_command("help", channel_id, _args, _team_id) do
+    send_message_to_channel(channel_id, @help_text)
+  end
+
+  defp handle_command("create", channel_id, args, team_id) do
+    %{
+      "channel" => %{
+        "name" => channel_name,
+        "topic" => %{"value" => topic}
+      }
+    } = get_channel_info(channel_id)
+
+    name = if is_nil(Enum.at(args, 0)), do: channel_name, else: Enum.join(args, " ")
+
+    cond do
+      has_hubs_url(topic) ->
+        send_message_to_channel(
+          channel_id,
+          "A Hubs room is already bridged in the topic, so I am cowardly refusing to replace it."
+        )
+
+      String.length(name) > 64 ->
+        send_message_to_channel(
+          channel_id,
+          "Room name is too long (over 64 characters). Please shorten room name."
+        )
+
+      true ->
+        {:ok, %{hub_sid: hub_sid} = new_hub} = Hub.create_new_room(%{"name" => name}, true)
+        update_topic(channel_id, add_hub_topic(topic, Hub.url_for(new_hub)))
+
+        HubBinding.bind_hub(%{
+          "hub_id" => hub_sid,
+          "type" => "slack",
+          "community_id" => team_id,
+          "channel_id" => channel_id
+        })
+    end
+  end
+
+  defp handle_command("remove", channel_id, _args, _team_id) do
+    %{"channel" => %{"topic" => %{"value" => topic}}} = get_channel_info(channel_id)
+
+    if !has_hubs_url(topic) do
+      send_message_to_channel(channel_id, "No Hubs room is bridged in the topic, so doing nothing :eyes:")
+    else
+      update_topic(channel_id, remove_hub_topic(topic))
+    end
+  end
+
+  defp handle_command("", channel_id, _args, _team_id) do
+    send_message_to_channel(channel_id, @help_prefix)
+  end
+
+  # catches requests that do not match the specified commands above
+  defp handle_command(_command, channel_id, _args, _team_id) do
+    send_message_to_channel(channel_id, "Type \"/hubs help\" if you need the list of available commands")
+  end
+
+  defp has_hubs_url(topic) do
+    String.match?(topic, get_hub_url_regex())
+  end
+
+  defp remove_hub_topic(topic) do
+    # In Slack topic, hub url is surrounded by '<>' in slack ex: <https://etc>
+    # A topic set in slack with <> will not match regex. It's coded with '&gt;&lt;'
+    new_topic = Regex.replace(~r/[<>]/, topic, "")
```

Comment:
I think this will remove all occurrences of "<" and ">", regardless of where they are in the topic, which we probably don't want.